test/Network/MQTT/ParserEncoderSpec.hs

```diff
@@ -0,0 +1,61 @@
+{-# LANGUAGE FlexibleInstances, MultiParamTypeClasses #-}
+
+module Network.MQTT.ParserEncoderSpec       (spec) where
+
+import           Network.MQTT.Packet
+import           Network.MQTT.Encoder       (encodePacket)
+import           Network.MQTT.Parser        (parsePacket)
+
+import           Test.SmallCheck.Series
+import           Test.Hspec
+import           Test.Hspec.SmallCheck      (property)
+
+import           Data.Attoparsec.ByteString (parseOnly)
+import           Data.Word                  (Word16)
+import           Data.ByteString            (ByteString, singleton)
+import           Data.ByteString.Builder    (toLazyByteString)
+import           Data.ByteString.Lazy       (toStrict)
+import qualified Data.Text as T
+
+spec :: Spec
+spec = do
+  describe "Parser-Encoder" $ do
+    it "test identity" $ property $
```

Comment:
The first argument of `it` should be a statement. Something along the way with `parsePacket is a reverse of encodePacket` or `parsePacket . encodePacket = id`.